pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -285,6 +286,11 @@ RUN /opt/conda/bin/mamba install -y -c conda-forge \
   cmake \
   lit
 
+# Patches
+RUN pip install "pillow>=9.5"
+RUN /opt/conda/bin/mamba install -y -c conda-forge \
+  requests==2.31.0
```

Comment:
need to update requests as earlier installation keep requests with older version

Reply:
Would it still work if this was changed to `requests>=2.31` rather than `requests==2.31.0`? The former is preferred because it allows requests to be updated to newer versions if possible.

Reply:
+1 and lets update the requests package installed in the common section

Reply:
I see, let me try with quotes

Reply:
whats the issue, conda does not respect the pinning?

Reply:
using quotes worked